packages/xo-server-web-hooks/src/index.js

```diff
@@ -2,16 +2,48 @@ import { createLogger } from '@xen-orchestra/log'
 
 const log = createLogger('xo:web-hooks')
 
+function constructPayload(format, data, type) {
+  let payload
+
+  switch (format) {
+    case 'json':
+      payload = JSON.stringify({ ...data, type })
+      break
+    case 'office365':
+      {
+        // https://learn.microsoft.com/en-us/outlook/actionable-messages/message-card-reference
+        const facts = Object.keys(data).map(key => {
+          const value = data[key]
+          return { name: key, value: typeof value === 'string' ? value : JSON.stringify(value) }
+        })
+
+        payload = JSON.stringify({
+          '@type': 'MessageCard',
+          '@context': 'https://schema.org/extensions',
+          themeColor: 'BE1621',
```

Comment:
Where does this color come from? Shouldn't we use an XO color like `9B92FF` instead?

Reply:
This color come from the figma DS (`danger txt-base`)

Reply:
Ok, thanks. And why "danger" instead of "brand"?

Reply:
I think it would make more sense, yes. "danger" usually means that there was an error. "brand" would be more neutral and identify XO as the source of the notification.